src/Filter/AbstractDateFilter.php

```diff
@@ -119,12 +123,30 @@ public function getRenderSettings()
         ]];
     }
 
+    /**
+     * @param string $field
+     * @param array  $data
+     */
+    abstract protected function applyTypeIsLessEqual(ProxyQueryInterface $queryBuilder, $field, $data);
```

Comment:
Since we dropped php 7.1, you can use parameter type hints here.

Reply:
oh right, nice!